recipe-server/normandy/recipes/urls.py

```diff
@@ -32,4 +32,9 @@
         api_v1_views.ActionImplementationView.as_view(),
         name='action-implementation'
     ),
+    url(
+        r'^api/v2/identicon/(?P<generation>v[0-9]):(?P<seed>.{1,64}).svg',
```

Comment:
@mythmon Thoughts on an API endpoint with `.svg` at the end? I'm on the fence but don't have a great alternative.

Reply:
I'm ok with it. We aren't doing it here, but DRF can be configured to vary based on the extension of the API endpoint called. So in theory, `/api/v2/identicon/v1:foo.svg` could return the image, and `/api/v2/identicon/v1:foo.json` could return a JSON version of the data used to generate the svg. I think for non-json endpoints, this is a good idea.